string[] poi_names
---
bool found
string poi_name_out